apps/site/src/middleware.page/return-types-as-json/hardcoded-types.ts

```diff
@@ -1,7 +1,11 @@
+/*
+ @todo - We should be able to import from the `@blockprotocol/graph` package here but we're running into strange errors
+   with named exports, CommonJS, and ESM modules from the dependency on `/core`
+ */
 import {
   DATA_TYPE_META_SCHEMA,
   ENTITY_TYPE_META_SCHEMA,
-} from "@blockprotocol/graph";
+} from "@blockprotocol/type-system/slim";
```

Comment:
Doing this fixed _local_ site building which wasn't understanding the `@blockprotocol/core` package and erroring with:

```
Dynamic Code Evaluation (e. g. 'eval', 'new Function', 'WebAssembly.compile') not allowed in Edge Runtime 
Learn More: https://nextjs.org/docs/messages/edge-dynamic-code-evaluation
Import trace for requested module:
../../node_modules/es-module-lexer/dist/lexer.js
../../libs/@blockprotocol/core/dist/esm/html.js
../../libs/@blockprotocol/core/dist/esm/index.js
../../libs/@blockprotocol/graph/dist/es/main-non-temporal.js
./src/middleware.page/return-types-as-json/hardcoded-types.ts
./src/middleware.page/return-types-as-json.ts
```